packages/v4-sdk/src/utils/getBinIds.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Generate list of BinIds.
+ *
+ * @example
+ * given activeId = 100, numBins = 3, returns [99, 100, 101]
+ * given activeId = 100, numBins = 4, returns [98, 99, 100, 101]
+ */
+export const getBinIds = (activeId: bigint, numBins: bigint): bigint[] => {
```

Comment:
one more useful util function would be `getBinIdsByLiquidityDepths(activeId: bigint, depth: number)`.

```typescript
// depth is the liquidity depth in percentage
// means getting +/- 1% of liquidity from current bin
const binIds = getBinIdsByLiquidityDepth(100, 1)
```